index.php

```diff
@@ -9,34 +9,80 @@
   <!-- main posts loop -->
   <section id="posts" class="container">
 <?php
-if( have_posts() ) {
-  while( have_posts() ) {
-    the_post();
+  $categories = get_terms('category', array('orderby' => 'id', 'hide_empty' => false));
+
+  if ($categories) {
+    $i = 1;
+    foreach($categories as $category) {
```

Comment:
Well the modulus check can just be made to check if false and then start $i at 0. How does that `=>` function was really my question. I can guess what it does.